dodola/tests/test_core.py

```diff
@@ -115,6 +115,36 @@ def test_adjust_quantiledeltamapping_year_kind(variable_kind, expected):
     assert all(adjusted_ds[target_variable] == expected)
 
 
+def test_adjust_quantiledeltamapping_include_quantiles():
+    """Test that include-quantiles flag results in bias corrected quantiles
+    included in output"""
+    target_variable = "fakevariable"
+    n_simdays = 100 * 365  # 100 years of daily simulation.
```

Comment:
```suggestion
    n_simdays = 15 * 365  # 100 years of daily simulation.
```

I think this would speed up this test? Could prob go even lower?

Reply:
Definitely - I bumped it down to 5. Thanks for noticing that.